StandardCyborgNetworking/StandardCyborgNetworking/Model/ServerScene.swift

```diff
@@ -123,6 +123,49 @@ public struct ServerSceneVersion: Codable {
     }
 }
 
+public struct ServerSceneAsset: Codable {
+    
+    private enum CodingKeys: String, CodingKey {
+        case createdAt
```

Comment:
The camel casing vs. underscore casing is an unfortunate thing. I don't mind as long as things are consistent and work.

Reply:
Related: should this be `created_at`?

Reply:
I wanted to keep it consistent with `ServerSceneVersion`.